test/sqlsrv/sqlsrv_statement_cancel.phpt

```diff
@@ -94,7 +94,7 @@ Repro();
 --EXPECTREGEX--
 ﻿
 Test begins...
-\[Microsoft\](\[ODBC Driver 13 for SQL Server\]|\[ODBC Driver Manager\])([ ]{0,1}Function sequence error)
+(.*\[Driver Manager\]|\[Microsoft\](\[ODBC Driver 13 for SQL Server\]|\[ODBC Driver Manager\]))([ ]{0,1}Function sequence error)
```

Comment:
in Mac the error message is different from the others even though the real message `Function sequence error` is the same in all platforms